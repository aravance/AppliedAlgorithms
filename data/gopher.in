1 1.000 1.000 2.000 2.000
1.500 1.500

2 2.000 2.000 1.000 1.000
1.500 1.500
2.500 2.500

1 -5000.000 5000.000 -10000.000 -10000.000
0.000 0.000

1 -5000.001 5000.000 -10000.000 -10000.000
0.000 0.000

1 -5000.000 5000.000 -10000.000 -9999.999
0.000 0.000

1 -5000.000 0.000 10000.000 -10000.000
0.000 10000.00

1 5000.000 0.000 -9999.999 -10000.000
0.000 10000.00

1 -5000.000 -0.001 10000.000 -10000.000
0.000 10000.00

2 0.000 0.000 1000.001 0.000
333.334 0.000
-1000.001 0.000

2 0.000 0.000 1000.001 0.000
333.333 0.000
-1000.001 0.000

2 0.000 0.000 1000.001 0.000
333.334 0.000
-1000.002 0.000

10 1.000 2.000 3.000 -4.000
0.000 10.000
0.000 100.000
0.000 1000.000
1.000 1.000
0.000 -100.000
0.000 -1000.000
-10.000 -10.000
-10.000 10.000
100.000 100.000
100.000 -100.000

1000 0.000 0.000 7.000 7.000
243.379 -897.554
-1253.830 348.081
685.345 482.762
-747.461 1266.151
1006.980 450.433
352.048 -488.045
9379.248 1450.200
-61.665 -732.676
-181.580 509.649
-22.947 410.796
-217.637 -1625.487
-189.665 214.457
1032.073 -396.266
584.760 -233.622
10000.000 -776.757
712.810 -66.357
-275.975 199.281
10000.000 -370.372
-629.975 -51.227
-491.708 103.232
1331.291 -136.315
-2963.304 18.413
-330.210 -185.412
213.925 781.816
1624.235 -243.359
-2956.911 -291.091
454.843 7.922
536.950 -488.589
502.129 71.095
37.856 -10000.000
-204.618 425.572
10000.000 -694.967
401.587 -5841.364
-463.921 160.263
-268.354 403.486
368.219 238.050
-702.357 163.583
-695.327 2201.855
10000.000 68.463
-372.576 202.055
82.450 177.601
881.703 -365.782
-43.262 -2427.238
1207.367 -31.669
-1144.470 -426.065
1273.117 -590.509
727.667 -1904.096
-153.106 -1284.134
801.032 580.265
-720.619 -826.188
259.668 815.621
-16.591 -436.907
1540.883 1134.071
-582.253 -437.669
1325.901 429.954
215.696 -859.394
-255.626 676.351
-41.353 787.711
121.056 -1101.921
-48.120 -25.465
-679.504 -2192.733
1381.683 1270.477
-10000.000 44.572
-131.868 -678.731
4481.418 -2724.523
-13.414 177.064
75.379 502.965
302.493 -243.010
-611.216 -327.690
-8267.320 193.589
-452.646 -6558.043
-861.019 -752.844
75.806 -831.417
310.548 -1020.397
-610.220 793.461
4181.753 -10000.000
542.170 -771.618
444.363 -44.907
-191.920 -435.682
5458.403 -837.343
2.530 -1544.957
-270.101 -51.146
-110.291 830.569
305.189 -50.947
-2002.588 -716.776
3.584 -93.254
511.154 -10000.000
-10000.000 608.149
-413.480 -827.994
475.299 192.357
-56.975 1211.244
-54.838 4.313
1289.012 -719.986
60.661 3306.286
19.473 -2216.821
127.510 -556.208
1.921 -105.980
138.570 -526.983
-43.987 -3.452
432.775 -35.477
-645.773 -358.990
1438.447 -142.618
-1.804 300.460
-943.000 901.967
-1142.771 816.725
-824.593 75.866
-869.354 -1390.958
-9696.344 537.852
-2158.822 1120.003
653.833 388.338
-1118.875 -5152.763
-753.258 -329.442
1190.357 -2222.242
373.099 -1113.516
996.591 2173.028
463.431 168.203
-10000.000 -64.920
-151.446 672.574
-3283.144 161.428
-1693.317 -6717.088
62.137 852.681
-1345.710 -1931.190
-31.507 7786.071
1180.887 17.667
755.519 1116.770
116.466 106.681
183.097 -338.804
-18.877 -693.338
-105.286 -74.597
49.036 -480.067
613.149 -40.399
-17.782 191.476
89.231 358.615
-570.472 2727.893
576.086 -17.821
-1608.651 -297.395
373.121 -196.719
-412.602 -237.453
-1790.465 -1166.468
4708.054 5709.201
-347.496 60.342
1552.758 4654.246
32.982 -133.668
-1413.092 -506.026
357.986 -449.252
514.583 -259.732
-1180.225 -569.948
135.505 678.434
-284.959 536.857
311.929 -3103.262
-637.121 -120.320
-1342.098 531.849
-10000.000 -4809.168
-248.481 2190.137
-1717.594 -976.259
-1192.473 -926.215
-843.356 232.111
1340.426 -136.395
-920.821 -1491.721
-21.529 1195.317
-31.503 -1051.243
-270.456 -55.302
-180.734 -1330.915
-376.406 151.899
1083.482 -1316.991
4.566 -74.513
2997.634 282.405
104.965 63.968
-964.744 190.272
379.936 21.733
-33.941 288.224
631.027 -946.585
155.474 -1512.119
732.051 928.175
-10000.000 -284.899
170.370 -6898.026
53.207 -5113.265
464.991 3.715
33.658 -20.078
-25.220 -62.428
-157.971 208.721
-135.731 -24.069
515.906 164.897
-387.241 -825.734
1.729 346.392
372.046 -938.761
-483.264 535.000
-81.370 1.976
347.858 1493.002
-1747.450 -610.525
-884.607 -279.811
252.268 249.448
-199.149 -508.307
-2027.940 -1146.188
330.283 6251.507
-905.940 546.273
933.981 -675.561
-290.197 -18.564
51.220 -484.835
732.729 -198.698
985.377 -4.186
-6755.974 -733.165
-404.284 -294.065
-42.376 -25.290
-10000.000 -2.108
213.252 300.600
-316.731 148.927
-198.899 -395.915
-1898.590 -7865.060
-509.849 19.787
-564.768 -9204.355
-1069.479 -64.059
592.869 -734.335
446.994 241.076
907.147 947.765
-506.957 4077.013
263.424 -472.184
149.912 952.364
-138.754 1005.510
1191.952 10000.000
-301.296 -69.958
-258.826 -1174.161
52.308 -235.147
-300.135 10000.000
-302.808 -785.921
84.191 -454.007
-133.990 10000.000
129.754 -401.699
733.437 84.936
104.575 2780.510
-2344.915 -537.011
10000.000 66.585
219.186 757.251
-406.046 -112.016
-668.204 -669.488
1514.314 144.732
-5533.853 710.806
483.571 -992.407
1281.020 366.469
410.933 -675.139
-363.841 -247.395
-193.885 487.513
1423.726 387.659
-335.895 276.229
362.372 -843.680
-1070.528 -6093.224
-465.260 -232.673
-1603.902 46.864
-22.770 785.486
-48.682 80.253
-49.474 -113.930
-10000.000 -549.819
-417.313 -607.352
-76.469 -57.701
242.208 -444.296
10000.000 10000.000
-574.925 -1000.554
33.241 373.389
135.527 40.301
12.662 -10000.000
-911.458 238.973
-611.727 -956.071
-17.754 123.429
368.845 180.931
-212.801 -297.017
489.097 917.535
-4360.979 19.464
43.554 240.511
2046.300 69.685
4677.786 -480.966
370.740 171.655
-919.155 -7.066
-250.195 -5910.287
2927.826 36.977
-278.743 472.365
847.632 942.054
2118.561 214.743
52.083 38.015
-320.793 5465.489
342.254 -2353.154
433.464 -180.134
246.183 -2617.375
-485.756 1036.098
-185.760 -2276.393
865.257 563.219
-772.095 958.108
-605.214 8.707
-1943.186 91.104
667.865 4228.108
332.713 135.404
-1801.999 -11.240
2959.848 3931.317
-446.044 -216.075
450.493 -4.809
674.945 -177.025
0.616 -504.733
605.725 0.384
618.035 -587.108
-26.507 -107.163
-592.185 -731.978
-822.381 48.736
79.324 -565.476
-225.130 461.723
143.549 -310.273
-56.380 -216.147
181.868 115.336
1575.081 1131.221
122.259 -4109.307
-10000.000 -27.080
-916.405 398.241
1240.102 -357.143
663.850 2439.930
-4039.005 1401.907
548.853 187.346
-2657.235 14.121
138.089 338.563
-148.674 429.163
-34.107 1105.480
-1213.769 439.752
3466.274 10.553
990.062 705.590
-589.689 783.213
8.538 -2881.847
-8159.193 41.440
-1211.129 -389.390
-100.517 182.353
1156.596 112.827
244.134 11.998
-62.800 -908.585
-58.967 1060.724
-601.279 -10000.000
5285.055 -128.421
1880.409 -1095.952
1010.247 297.426
-5918.924 180.819
-809.566 -970.841
-10000.000 -10000.000
281.942 -557.462
-237.902 -186.863
5.596 -359.281
-7.629 -23.147
515.125 -203.623
690.734 357.058
309.469 -589.650
815.810 -886.566
-10000.000 -235.767
-31.813 -554.446
-551.264 756.619
-594.227 24.461
-371.611 326.336
234.027 253.580
7.488 293.757
609.424 158.240
-761.593 88.548
-688.848 -34.427
-12.176 -886.481
270.041 1433.058
1174.737 -124.664
-24.392 -3532.347
9431.660 -161.057
-106.558 246.466
296.231 -1058.312
-356.032 51.066
563.485 798.860
466.779 -56.395
-541.851 78.995
552.288 250.208
-485.817 -348.003
621.389 -1153.119
-74.445 -41.693
811.304 -59.840
-5.075 -13.555
-238.109 283.920
-18.949 1294.945
69.920 -1963.837
115.120 130.959
1196.242 96.915
-10000.000 -345.503
-680.753 541.616
-1.924 -4695.201
1037.705 -36.479
-574.822 -529.509
-76.351 848.695
-253.447 388.180
16.563 395.058
168.337 255.422
726.114 664.460
-144.657 -1083.530
1546.133 -751.364
-597.081 718.218
-542.855 149.394
-3034.367 78.945
428.669 -20.730
3.198 -5.127
-515.490 -300.152
-2454.885 -973.268
-606.381 348.273
-962.346 2360.736
397.179 -529.164
217.047 -164.143
804.104 -247.200
-24.551 -441.818
1497.992 -55.788
-1683.994 223.604
-4823.302 811.885
-204.912 293.169
-827.367 -745.361
556.664 -390.055
-231.561 79.416
1000.937 742.409
-158.290 2199.324
3.257 -147.287
-348.126 -174.046
-36.682 -781.524
-8356.805 0.330
518.443 523.116
-161.087 639.004
3042.068 -75.826
53.896 1318.349
10000.000 9.928
116.278 -268.718
-33.288 152.374
157.260 382.408
-556.750 -755.870
-18.595 -551.710
-321.004 5401.531
-815.412 -445.510
10000.000 -302.762
45.722 51.052
-128.536 7624.477
-721.017 -249.917
-1114.913 -123.508
12.221 -3272.448
22.953 -46.806
-49.342 709.833
-453.219 807.580
16.054 -318.175
742.617 -63.226
-1405.186 5.734
714.861 489.732
238.526 -176.947
-24.013 -40.361
-55.313 -53.725
-249.767 745.330
14.655 7.957
-581.952 -264.292
82.039 1308.850
-27.025 46.666
-402.680 588.523
528.215 -38.490
-245.899 51.925
204.495 1597.440
3369.505 113.488
526.045 -24.356
619.612 -256.409
-916.297 28.068
1031.129 1999.869
-213.699 -182.518
-1228.825 1842.284
743.772 783.644
115.869 1829.464
-201.757 -3663.922
-1685.028 -3277.146
561.361 -53.767
304.445 5611.062
235.106 -1688.625
318.616 46.496
21.906 -436.627
1685.207 1194.966
-1474.376 147.515
1014.938 -46.472
-2054.453 -561.896
-497.243 12.348
-3316.993 2490.957
-272.998 91.175
-288.405 37.069
137.532 481.089
-1481.130 -607.363
41.726 -541.015
754.066 981.888
-25.748 183.581
-1594.575 -22.971
486.092 -170.645
422.941 -232.983
-41.175 8871.405
-620.680 42.820
-75.952 1213.387
-261.993 -265.169
-248.705 -98.431
286.567 10000.000
-87.051 -545.233
-7387.432 135.052
-38.832 -52.414
-5887.656 -2261.913
202.743 -208.860
-185.735 -568.675
-1325.154 1917.479
-1598.461 408.349
-652.431 953.108
349.333 -956.428
2792.073 2241.656
495.053 28.247
1073.476 252.965
-70.035 431.729
1202.015 -8.307
126.257 -257.494
-2685.109 311.987
381.371 -1825.920
-817.821 -509.115
-893.018 430.608
-612.894 -37.604
-1515.190 514.345
-12.376 647.983
-595.514 -152.593
-725.283 -191.995
-49.360 -11.236
-718.678 -15.151
235.102 725.420
-115.091 -636.562
-324.926 7.978
-1631.167 30.726
-590.313 -67.462
470.550 -54.828
106.496 -9.935
617.255 -1266.158
-20.408 -195.937
2675.198 -357.724
423.846 2491.265
-544.255 90.770
-594.009 1086.647
329.887 -319.754
-94.709 264.471
-3135.048 -3729.588
-10000.000 -120.709
-181.546 -447.468
-4445.066 -0.110
721.625 1362.211
203.741 -364.922
-73.882 -4.611
-678.482 -167.396
28.667 -1227.669
191.879 492.787
-1059.689 -5817.699
-163.177 460.904
-6.531 202.348
442.162 -480.724
4279.795 -634.133
-100.614 -473.864
-335.251 345.012
-250.114 -248.033
4.933 3751.875
-1363.111 -938.485
2190.451 -54.375
-1213.679 -650.515
970.593 11.540
43.805 -196.873
-3385.946 289.616
-517.660 119.332
-238.641 -57.533
30.603 744.951
-1086.488 191.664
-243.299 4.043
-194.468 -15.127
-333.645 98.617
-154.664 -482.566
109.995 395.076
-851.514 335.220
-613.268 685.056
-899.545 -41.534
-143.826 623.278
-1426.789 3905.260
-139.708 10000.000
268.817 -42.478
586.376 1966.850
1294.774 -567.168
891.229 2705.447
736.349 40.147
39.453 66.354
-1263.734 267.389
10000.000 1333.702
110.733 277.756
52.503 820.664
1395.172 -1268.140
6514.398 273.642
-1287.690 311.815
388.774 251.434
10.037 -384.714
-25.085 197.919
-140.263 332.437
-364.499 4050.399
-3737.108 357.449
-584.136 1461.188
-524.841 247.144
97.192 212.995
176.845 -162.360
-1138.140 -6485.107
-102.239 2249.246
-413.380 -50.567
54.465 353.928
-647.845 -212.382
70.121 444.459
30.907 -121.117
-1006.504 243.674
1012.381 -288.238
-69.396 6609.549
58.985 -454.903
84.811 -908.581
-10.549 2621.785
9265.737 595.602
1195.554 46.445
1405.569 436.813
1344.495 4602.885
767.291 -69.470
-4724.534 374.369
1736.326 -396.034
585.499 -203.007
187.955 -4035.184
10000.000 -144.469
1895.497 -597.143
44.198 3.922
530.111 -3309.467
-549.509 278.934
3103.519 20.516
4342.330 46.597
-548.219 421.322
47.698 2062.854
-491.837 10000.000
2761.600 -2830.856
-1.595 778.495
321.683 -197.253
890.149 -461.907
-1144.611 122.521
-199.816 -741.279
10000.000 109.178
-830.049 -10000.000
-405.515 -191.928
2211.593 5431.595
102.647 -209.512
-854.790 -358.425
10000.000 87.076
34.700 198.668
-476.406 4.204
-994.949 2689.375
10000.000 -805.426
357.100 -303.846
374.028 -436.272
-868.658 145.191
559.674 708.881
-1564.990 649.498
-1145.933 432.480
12.891 395.421
13.103 -487.271
-487.231 -473.963
-1755.066 -2477.441
-341.530 366.032
-393.122 1278.432
183.367 476.171
-43.577 -8769.977
35.043 606.032
198.780 1069.742
75.284 -91.979
-76.079 1162.410
-344.179 1666.176
1004.173 -310.242
-10000.000 298.219
-1301.942 -1737.802
-817.983 5.996
-87.853 650.237
-83.800 -402.794
419.087 -9485.040
-778.957 10000.000
-459.534 37.892
1694.442 -2186.126
-1018.140 183.544
234.988 -127.720
-311.727 -131.543
252.809 2413.925
-6273.837 505.994
-805.720 -1098.735
2095.640 -233.280
-10000.000 -672.677
-770.169 -5408.233
140.825 -828.749
-161.817 -54.300
-3425.318 -212.038
-191.167 -707.071
-272.720 383.937
952.891 612.252
68.865 8.720
-356.503 -993.390
-619.450 3550.138
2189.278 604.864
1792.751 673.781
70.794 -281.379
-271.333 -241.384
477.620 52.159
36.727 -93.260
-4612.133 -1614.747
-456.915 222.591
191.133 231.584
-1656.248 -25.268
730.597 -2808.936
1016.911 375.292
2200.925 305.886
231.732 -446.089
541.761 -2808.475
-3677.121 -1.725
-310.377 874.437
-678.888 2480.230
15.522 -765.024
191.438 691.941
994.156 827.982
2451.059 338.996
465.010 -338.150
139.054 -188.679
355.063 203.593
98.738 437.624
1793.996 -966.100
24.568 1734.770
-348.971 70.550
1052.773 1723.731
-17.150 -31.374
708.029 -59.320
-164.252 -851.226
600.996 372.264
-419.018 -199.420
636.690 400.318
114.706 -1142.537
-468.120 102.305
325.657 1315.752
-626.797 -552.059
831.393 -261.296
94.581 478.969
81.252 505.696
174.696 373.827
-1457.749 10000.000
-7143.330 -633.024
508.652 -10000.000
4478.074 -608.257
-1907.284 631.514
-833.747 -2487.628
-378.468 -1799.145
2509.973 -662.924
304.625 -547.503
-1892.258 -5097.348
-429.100 128.155
2638.125 66.705
989.385 -587.495
-52.934 128.662
399.080 -43.008
253.078 1556.290
-302.601 10000.000
816.302 689.002
39.914 -530.121
-348.815 -30.674
-13.030 -47.069
-561.842 -8027.715
1097.930 1639.782
2094.630 -10000.000
850.704 30.115
3723.664 -121.438
50.227 -2473.745
-10000.000 569.398
-478.238 -1360.594
-3327.695 206.694
856.026 383.798
-2150.018 460.578
-98.551 138.662
-46.321 -492.789
-181.445 -39.627
-1096.696 70.293
43.062 -100.235
-143.038 42.720
-2000.365 439.594
-927.959 52.687
-306.343 382.867
639.723 1327.690
-1431.413 -110.587
22.121 -136.869
-156.764 -370.747
910.375 -82.127
677.250 687.347
-829.145 -305.891
523.054 1669.985
35.517 94.072
818.236 441.721
394.014 -494.195
-217.484 -1652.373
-6.785 -116.900
16.941 -548.703
-236.324 -531.258
749.825 -1833.515
-219.256 46.389
-7.003 185.340
-23.038 545.122
1432.243 699.742
-262.378 2.898
10000.000 483.142
-478.753 8.384
111.588 1755.840
-159.438 -811.228
-212.352 -633.326
284.072 388.750
43.218 -3139.799
270.638 1735.094
2423.563 -1696.548
-70.369 381.727
-5.790 -222.694
236.203 410.511
0.305 162.234
76.046 -201.740
3.167 183.600
172.253 -98.702
1381.174 11.034
-1918.053 -6777.140
-3.234 204.630
-2285.152 2338.364
0.000 2138.352
-24.765 2509.798
1676.335 78.469
-3943.099 1654.976
-249.656 17.681
-39.730 2477.158
2399.326 678.765
-391.637 1369.257
99.660 -250.759
-416.820 -1866.228
293.880 102.665
385.446 -665.305
1020.422 -110.936
2586.846 30.076
-179.269 317.585
-1113.091 -1504.215
-783.692 -565.826
-283.691 484.352
776.722 11.180
627.816 -105.813
-274.023 -775.400
734.029 -1193.082
-42.699 178.873
-51.497 -284.017
446.276 138.961
290.643 5.655
631.465 1309.515
-109.006 -1540.363
523.476 371.326
47.299 -564.614
178.423 369.112
-45.509 -284.071
-233.926 53.719
310.504 502.203
1638.987 119.902
-62.216 -1407.488
454.577 896.477
-95.313 -1142.717
98.812 -241.379
550.613 -18.827
4182.108 -7245.849
-732.863 252.726
-924.421 -235.605
-695.292 -656.399
2628.833 -3202.028
-1757.291 -535.601
104.472 259.888
1070.484 -30.791
575.826 1253.836
430.905 -659.187
-1734.846 330.739
51.066 -237.089
4826.755 134.076
402.659 1407.219
-1039.692 -1298.000
629.603 49.738
-14.236 -555.845
502.315 -10000.000
-182.608 -1006.315
5588.234 965.056
590.610 -1410.663
276.509 -19.397
-115.893 385.283
-4756.976 -578.278
-676.920 4279.076
-1499.526 530.765
273.082 -691.791
213.719 -5937.581
-149.671 -98.937
21.641 566.118
-2353.524 1031.190
-2472.569 490.173
-174.982 766.888
-1513.287 116.512
183.076 -1.194
341.410 925.594
675.946 -16.382
-1277.050 -550.520
-2523.023 329.655
10000.000 -574.908
92.498 445.708
-212.283 -10000.000
-500.805 -125.673
1701.799 9654.917
-203.885 -971.307
-262.132 -376.806
-52.684 705.595
-605.062 -10000.000
2090.886 94.997
2603.852 -20.562
602.965 -23.011
20.423 824.742
614.978 -228.978
-67.667 -288.475
1433.700 3812.948
-10000.000 -91.998
2282.965 1647.634
316.665 -10000.000
-1565.002 -72.549
243.189 100.716
1041.524 279.943
2161.365 -755.683
-3484.366 -392.130
0.504 -408.714
-476.769 -148.463
-828.090 -294.191
289.577 7608.950
522.970 20.554
-1755.187 1336.773
141.875 -44.404
253.316 -505.871
56.749 -4649.264
-146.851 5955.029
1190.650 -481.194
-1194.181 -116.756
-1565.473 272.944
81.721 197.559
-419.256 15.525
-3931.276 -367.145
1422.049 721.460
16.297 -45.640
205.168 269.870
418.069 498.936
-10000.000 -206.599
2317.705 -413.058
-323.632 -1612.557
354.489 164.621
62.374 -257.883
178.196 -10000.000
-320.051 -1273.791
-1156.583 175.899
228.002 718.947
-1885.048 3257.368
-640.096 1455.046
-268.309 725.532
-191.694 3417.070
677.264 29.314
-244.795 2391.961
-4.848 -76.788
-304.563 703.049
-10000.000 209.275
740.709 -1814.234
201.076 -337.085
3.056 -493.877
-594.254 4163.942
-1668.887 42.705
113.505 -1643.655
-647.902 -279.810
-660.038 669.331
138.412 -244.976
71.674 8305.333
93.823 -3156.274
-3911.217 263.299
515.912 -151.931
-10000.000 55.047
1473.042 -2053.026
63.079 -183.082
31.469 1911.033
-4.407 344.995
1286.662 -513.932
-300.380 -10000.000
165.233 -1069.163
-574.458 295.251
-766.976 -214.155
1702.953 -1705.129
-332.145 80.303
-1357.230 -145.696
-888.310 -1786.180
-123.401 -2.254
148.272 651.598
3743.060 -402.152
910.794 -614.592
-1043.268 262.233
-1002.079 -969.883
-6263.263 -193.411
7245.204 -704.532
-83.259 502.260
227.463 325.014
573.191 1178.333
-477.171 403.476
258.307 -3103.571
192.723 -558.266
49.569 1677.317

0 0.000 0.000 0.000 0.000
