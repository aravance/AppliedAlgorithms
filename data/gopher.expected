The gopher cannot escape.
The gopher can escape through the hole at (2.500,2.500).
The gopher can escape through the hole at (0.000,0.000).
The gopher cannot escape.
The gopher cannot escape.
The gopher can escape through the hole at (0.000,10000.000).
The gopher cannot escape.
The gopher cannot escape.
The gopher can escape through the hole at (-1000.001,0.000).
The gopher can escape through the hole at (333.333,0.000).
The gopher cannot escape.
The gopher can escape through the hole at (1.000,1.000).
The gopher can escape through the hole at (3.198,-5.127).
